src/js/stores/ChronosStore.js

```diff
@@ -0,0 +1,117 @@
+import {EventEmitter} from 'events';
+
+import AppDispatcher from '../events/AppDispatcher';
+import ChronosActions  from '../events/ChronosActions';
+import {
+  CHRONOS_JOBS_CHANGE,
+  CHRONOS_JOBS_ERROR,
+  VISIBILITY_CHANGE
+} from '../constants/EventTypes';
+import Config from '../config/Config';
+import JobTree from '../structs/JobTree';
+import {
+  REQUEST_CHRONOS_JOBS_ERROR,
+  REQUEST_CHRONOS_JOBS_ONGOING,
+  REQUEST_CHRONOS_JOBS_SUCCESS,
+  SERVER_ACTION
+} from '../constants/ActionTypes';
+
+import VisibilityStore from './VisibilityStore';
+
+let requestInterval;
+
+function startPolling() {
+  if (!requestInterval) {
+    ChronosActions.fetchJobs();
+    requestInterval = global.setInterval(
+      ChronosActions.fetchJobs,
+      Config.getRefreshRate()
+    );
+  }
+}
+
+function stopPolling() {
+  if (requestInterval) {
+    global.clearInterval(requestInterval);
+    requestInterval = null;
+  }
+}
+
+function handleInactiveChange() {
+  let isInactive = VisibilityStore.get('isInactive');
+  if (isInactive) {
+    stopPolling();
+  }
+
+  if (!isInactive && ChronosStore.shouldPoll()) {
+    startPolling();
+  }
+}
+
+VisibilityStore.addChangeListener(VISIBILITY_CHANGE, handleInactiveChange);
+
+class ChronosStore extends EventEmitter {
+  constructor() {
+    super(...arguments);
+
+    this.data = {
+      jobTree: {id: '/', items: []}
+    };
+
+    // Handle app actions
+    this.dispatcherIndex = AppDispatcher.register(({source, action}) => {
```

Comment:
I added the example as I wanted to illustrate that we do have more urgent problems with inconsistency than a new store applying the new ES6 style guide. As I wrote, I thought that the prefered rollout strategy for the ES6 classes and such is to implement new components and stores(!) following the ES6 style guide. Thus I didn't thought that there is anything wrong with this store.

I'll estimate the effort to update all the store and use that estimate to find a good solution for this "inconsistency". 

No matter how we're proceeding with this store, I'd like you to add an contribution file or something similar to document the latest decisions on how to rollout he latest ES6 features, and your plan on how to take care of all the inconsistency that is already present, as we probably wasted time doing this store in the new ES6 style and I like us to prevent this in the future.